sample/HttpTrigger-Identities/index.js

```diff
@@ -0,0 +1,17 @@
+﻿module.exports = function (context, req) {
+    context.log('Node.js HTTP trigger function processed a request. There are %s identities associated with this request', context.bindingData.identitiesArray.length);
```

Comment:
Can you make this sample perform the same operations and return that the CSharp example does?